app/models/team.rb

```diff
@@ -0,0 +1,21 @@
+class Team < ActiveRecord::Base
+  has_many :team_members
+  has_many :users, -> { select("users.*, team_members.accepted AS team_members_accepted") },
+                   through: :team_members
+  has_one :entry
+  has_one :owner, class_name: "User", primary_key:"owner_id", foreign_key: "id"
+  
+  validates :name, :uniqueness => true
+  
+    # Convert team name to friendly url format
+  def slug
+    name.downcase.gsub(' ', '-').parameterize
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.